services/net/reporting/ReportingManager.cs

```diff
@@ -534,6 +534,17 @@ private async Task GenerateAndSendReportAsync(ReportRequestModel request, API.Ar
         await SendReportAsync(request, report, instanceModel, sectionContent);
     }
 
+    /// <summary>
+    /// Determine if this request is to resend the report.
+    /// </summary>
+    /// <param name="request"></param>
+    /// <param name="instance"></param>
+    /// <returns></returns>
+    private bool IsResend(ReportRequestModel request, API.Areas.Services.Models.ReportInstance.ReportInstanceModel? instance)
```

Comment:
This logic allows for a report to be sent out without needing to fetch all the content and regenerating the output.